tests/test_cache_wrapper.py

```diff
@@ -0,0 +1,46 @@
+import unittest
+import sys
+from pathlib import Path
+
+# Add the project root to Python path
+project_root = str(Path(__file__).parent.parent)
+sys.path.insert(0, project_root)
```

Comment:
Should remove this. If the working directory when running the tests is at the root of the repo, it should just pick up the `mlx_engine` import